pkg/snapshot/restorer/restorer.go

```diff
@@ -123,19 +124,27 @@ func (r *Restorer) restoreFromBaseSnapshot(ro RestoreOptions) error {
 
 	walDir := filepath.Join(memberDir, "wal")
 	snapdir := filepath.Join(memberDir, "snap")
-	if err = makeDB(r.zapLogger, snapdir, ro.BaseSnapshot, len(cl.Members()), r.store, false); err != nil {
+	if err = r.makeDB(r.zapLogger, snapdir, ro.BaseSnapshot, len(cl.Members()), r.store, false); err != nil {
 		return err
 	}
-	return makeWALAndSnap(r.zapLogger, walDir, snapdir, cl, ro.Config.Name)
+	return r.makeWALAndSnap(r.zapLogger, walDir, snapdir, cl, ro.Config.Name)
 }
 
 // makeDB copies the database snapshot to the snapshot directory.
-func makeDB(logger *zap.Logger, snapdir string, snap snapstore.Snapshot, commit int, ss snapstore.SnapStore, skipHashCheck bool) error {
-	rc, err := ss.Fetch(snap)
+func (r *Restorer) makeDB(logger *zap.Logger, snapdir string, snap snapstore.Snapshot, commit int, ss snapstore.SnapStore, skipHashCheck bool) error {
+	reader, err := ss.Fetch(snap)
```

Comment:
Any particular reason to change the variable name from `rc` to `reader` here?

Reply:
no big reason ....I will change it back to `rc `

Reply:
no big reason .... changed it back to `rc`